### PR TITLE
login page

### PR DESCRIPTION
**Pull request by shivanigangadharan**

### CODE REVIEW

File: CSS/login.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+body{
+     background: var(--lightergrey);
+     display: flex;
```
Comment (SanjayJatti) [inline CSS/login.css]: You can create utility class for flex-row, flex-column, flex-center, etc.

File: CSS/login.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+}
+.remember-me{
+    display: flex;
+    justify-content: space-between;
+}
+.remember-me a{
+    color: var(--blue);
+    text-decoration: none;
+}
+.btn-login{
```
Comment (SanjayJatti) [inline CSS/login.css]: You can use button class  your from component library

**Review by SanjayJatti — APPROVED** [review]
Other than suggestion, code is clean.

**Review by Manpreet-hub — APPROVED** [review]
Looks good!

File: CSS/login.css (hunk 1 above)
Comment (shivanigangadharan) [reply CSS/login.css]: Moved this part of code from `body` to a class. 👍

File: CSS/login.css (hunk 2 above)
Comment (shivanigangadharan) [reply CSS/login.css]: Done in commit [9d8bc9a](https://github.com/shivanigangadharan/e-mart/pull/1/commits/9d8bc9a92d00c428933f88f8425ca1ae6c91de9d).

**Pull request merged**